### PR TITLE
feat: update issues content

### PR DESCRIPTION
**Pull request by nikromen**
example how issues will look like: https://github.com/nikromen/playground/issues/10

### CODE REVIEW

File: script.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 BODY_TEMPLATE = "Original {what}: {link}\n" "Opened: {date}\n" "Opened by: {user}"
 PAGURE_USERNAME = "INSERT_USERNAME"
+DESCRIPTION_TEMPLATE = "\n\n`with description:`\n{description}"
```
Comment (FrostyX) [inline script.py:15]: I would drop the "with description" text and just write the description itself (and the newlines).

File: script.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
...
+COMMENT_TEMPLATE = "\n\n---\n\n`{user} commented at {date}:`\n{comment}"
```
Comment (FrostyX) [inline script.py:16]: What do you think about putting the comment header into `<h4>`?

    #### [{user}](https://accounts.fedoraproject.org/user/{user}) commented at {date}:

I like the rest of the format.

**Review by FrostyX — COMMENTED** [review]
A few notes but otherwise LGTM.
But I would like to wait till @praiskup reviews as well.

File: script.py (hunk 2 above)
Comment (nikromen) [reply script.py:16]: yep, this looks better IMO

**Comment by praiskup** [comment]
Thanks for the work.  Don't you have a "real" example?  The example issue refers to issue 2019, but it contains "stub" comments.

**Comment by praiskup** [comment]
Looks nice. +1

**Comment by nikromen** [comment]
just updated the script with assertion check (check pagure and GH id)

**Comment by nikromen** [comment]
real examples: https://github.com/fedora-copr/copr/issues/2363 https://github.com/fedora-copr/copr/issues/2362 https://github.com/fedora-copr/copr/issues/2364

**Comment by nikromen** [comment]
so because pagure counts the metadata stuff (info about closing issue etc.) as a normal comment (wtf) I added a dumb condition to ignore this

**Comment by praiskup** [comment]
+1

**Pull request merged**